definitions/divorce/json/AuthorisationComplexType/AuthorisationComplexType-roles-and-permissions-resp-journey-nonprod.json

```diff
@@ -0,0 +1,34 @@
+[
+  {
+    "LiveFrom": "26/01/2021",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "RespondentOrganisationPolicy",
+    "ListElementCode": "OrgPolicyCaseAssignedRole",
+    "UserRole": "[RESPSOLICITOR]",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "26/01/2021",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "RespondentOrganisationPolicy",
+    "ListElementCode": "OrgPolicyReference",
+    "UserRole": "[RESPSOLICITOR]",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "26/01/2021",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "RespondentOrganisationPolicy",
+    "ListElementCode": "Organisation",
+    "UserRole": "[RESPSOLICITOR]",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "26/01/2021",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "RespondentOrganisationPolicy",
+    "ListElementCode": "Organisation.OrganisationID",
```

Comment:
is this really going to work?
I thought you'd have to create another complex type for this...

Reply:
Sorry, do we authorisation for complex types? That's new, right?

Reply:
Yep apparently those are authorisations that can be applied to each different field inside a complex type.